packages/paste-libraries/react-textarea-autosize/src/hooks.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import useLatest from 'use-latest';
+
+export {default as useComposedRef} from 'use-composed-ref';
+
+export const useWindowResizeListener = (listener: (event: UIEvent) => any): any => {
```

Comment:
maybe rename this file to `useWindowResizeListener` because we have another hook as its own file. Can move them to a hooks folder if you really want